src/main/java/com/google/sps/data/User.java

```diff
@@ -34,10 +34,10 @@ public String toString() {
   } 
 
   public ProfileType toEnum(String input) {
-      if(input == "student" || input == "STUDENT") {
-          input = "STUDENT";
+      if(input == "Mentee" || input == "MENTEE") {
```

Comment:
nit: [L37 and L40]: Maybe input.equalsIgnoreCase("STUDENT") ? More succinct and you also catch "Student".

Reply:
It won't be STUDENT ever because the usertype coming from the HTML is Mentee or Mentor

Reply:
Done